ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -18337,6 +18337,25 @@ void HqlCppTranslator::doBuildNewRegexFindReplace(BuildCtx & ctx, const CHqlBoun
     // Because the search instance is created locally, the search parameter is always going to be valid
     // as long as the find instance.  Only exception could be if call created a temporary class instance.
     bool cloneSearch = false;
+    CHqlBoundExpr boundTimer, boundStart;
+    if (options.timeRegex)
```

Comment:
The compilation of regex expression can take some time.  Not sure if that time is significant.  However, if this section is moved before doBuildRegexCompileInstance (line18322) then the timer accounts for the regex compile tome.

Reply:
That is true.  However the compile time would need to be recorded separately since it is generally done once in a different context.  If the regex is a constant and the search is done many times it is less significant.
It would potentially be useful to time the subgraph start/create time as well (possibly more so) since it would include the regex compile times.